nautobot/extras/models/groups.py

```diff
@@ -80,6 +109,11 @@ def members(self):
             pk__in=self.static_group_associations.values_list("associated_object_id", flat=True)
         )
 
+    @property
```

Comment:
Should we have a `set_members` helper function in addition to `add_members` and `remove_members`?